ideascube/mediacenter/forms.py

```diff
@@ -17,6 +18,9 @@ class Meta:
         widgets = {
             'lang': LangSelect,
         }
+        help_texts = {
+            'original': _('Maximum size : 500 Mo.'),
+        }
```

Comment:
Why not put the `help_text` on the model field?

I know it feels wrong... but it's the idiomatic Django way...